corehq/ex-submodules/casexml/apps/phone/tests/performance_tests.py

```diff
@@ -21,6 +21,11 @@
 REFERRAL_TYPE = 'referral'
 
 
+def setupModule():
```

Comment:
I think it doesn't matter in this case, but just found out yesterday that a `setupModule` function will cause an entire [`tests` package](https://github.com/dimagi/commcare-hq/blob/e704f8c37e650401e1479cc374150bbcc1c2a28a/corehq/ex-submodules/casexml/apps/phone/tests/__init__.py) to be skipped if there is a `from the.skipped.module import *` in `__init__.py`. The takeaway is to be sure to remove `from ... import *` in `__init__.py` (at least for the module that's being skipped). I have a TODO list item to remove all `*` imports at some point in the future after we've used nose for a while.